src/main/scala/mesosphere/marathon/api/TaskKiller.scala

```diff
@@ -28,15 +29,16 @@ class TaskKiller @Inject() (
   def kill(
     appId: PathId,
     findToKill: (Iterable[Task] => Iterable[Task]),
-    wipe: Boolean = false)(implicit identity: Identity): Future[Iterable[Task]] = {
+    wipe: Boolean = false,
+    killCount: Integer = -1)(implicit identity: Identity): Future[Iterable[Task]] = {
```

Comment:
The TaskKiller does not need to know anything about `killCount`.
`findToKill` is a function that chooses the tasks to kill and can be limited.
Please revert all changes to TaskKiller.